docker/lib/dependabot/docker/file_parser.rb

```diff
@@ -75,15 +79,17 @@ def parse
 
       private
 
+      sig { returns(T::Array[Dependabot::DependencyFile]) }
       def dockerfiles
         # The Docker file fetcher fetches Dockerfiles and yaml files. Reject yaml files.
         dependency_files.reject { |f| f.type == "file" && f.name.match?(YAML_REGEXP) }
       end
-
+      sig { params(parsed_from_line: T.untyped).returns(T.untyped) }
```

Comment:
Can you be more specific with the types here? Looks like it should be `params(parsed_from_line: T::Hash[String, String]).returns(String)`

Reply:
Fixed this as per review comments